src/main/scala/cognite/spark/v1/FlexibleDataModelBaseRelation.scala

```diff
@@ -274,6 +219,11 @@ abstract class FlexibleDataModelBaseRelation(config: RelationConfig, sqlContext:
             case s if s.equalsIgnoreCase("space") => n.space
             case s if s.equalsIgnoreCase("spaceExternalId") => n.space
             case s if s.equalsIgnoreCase("externalId") => n.externalId
+            case s if s.equalsIgnoreCase("metadata.cursor") => cursor.getOrElse("")
```

Comment:
Since "." is disallowed in all property ids (and spaces and views), the thinking is that metadata. as prefix is safe. Could of course use node or edge.

Reply:
oh, that is a good point!

Reply:
Out of the ones listed, the following are reserved:
lastUpdatedTime
deletedTime
createdTime

The ones that are not:
- version
- cursor

Since the first ones are reserved, should we not prefix those?